packages/xstate-graph/test/types.test.ts

```diff
@@ -1,11 +1,11 @@
 import { createMachine } from 'xstate';
-import { getMachineShortestPaths } from '../';
+import { getMachineShortestPaths } from '../src/shortestPaths';
```

Comment:
Fixes the TS errors when passing args to `getMachineShortestPaths`

Reply:
TS errors seem to sometimes be reported when running tests in watch mode
![CleanShot 2023-03-22 at 10 58 55](https://user-images.githubusercontent.com/279239/226946157-a81e3f83-2728-4803-b1b6-74916ffb79ff.png)